site/app/Resources/views/default/project_search.html.twig

```diff
@@ -8,22 +8,18 @@
 
 {% block stylesheets %}
   {{ parent() }}
-  {% stylesheets filter="cssrewrite"
-  "assets/css/instantsearch.min.css" %}
-  <link rel="stylesheet" href="{{ asset_url }}"/>
-  {% endstylesheets %}
 {% endblock %}
 
 {% block content %}
-  <form name="search-form" method="get" action="{{ path('librecores_project_repo_project_search') }}">
+  <form class="search-form" method="get" action="{{ path('librecores_project_repo_project_search') }}">
     <div class="librecores-search">
       <div class="row">
         <div class="col-sm-3">
           <h1>Search</h1>
         </div>
         <div class="col-sm-9">
           <div class="input-group" style="width:100%">
-            <input type="search" id="search-input" placeholder="Search for ..." class="form-control">
+            <div id="search-input" placeholder="Search for ..." class="form-control"></div>
```

Comment:
Why doesn't input work here?

Reply:
Instantsearch.js > 3 has stopped using `input` tags for widgets anymore. You can create your own custom inputs but adding a div certainly serves the purpose.